site/snippets/nav.php

```diff
@@ -13,9 +13,7 @@
   <?php endif; ?>
   <nav>
     <ul class="navigation">
-      <?php if ($page->hide_home_nav() && !$page->hide_home_nav()->toBool()): ?>
```

Comment:
if we are doing this, then we should remove the fields from the template `.yml` as well